supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

```diff
@@ -51,7 +51,7 @@
 [discrete]
 [[pc]]
 ==== image:images/yes.png[yes] PC (noun)
-*Description*: _PC_ is an abbreviation for personal computer. See the _IBM Style_ guide for further information.
+*Description*: _PC_ is an abbreviation for personal computer. For more information, see _IBM Style_.
```

Comment:
@adellape @IngridT1 Using "the _IBM Style_ guide" was actually correct (and that's how we refer to it consistently throughout the guide). @ndeevy is correct in that we changed instances of "the _IBM Style Guide_" to lowercase "guide" and move it outside of the italics.

Sorry to cause you more work @adellape, but would you mind reverting those 3 instances